api/src/test/java/com/flipkart/flux/api/WorkflowStateSummaryTest.java

```diff
@@ -10,24 +10,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.flipkart.flux.commons.dto;
+package com.flipkart.flux.api;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.IOUtils;
+import org.hamcrest.core.Is;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
```

Comment:
1. How does it improve readability ? Most IDEs hide all import statements in one single line any way. 
2. It doesn't create any linkages 

Please don't think i'm dragging on for the heck of it. It is important that we bring this to a logical conclusion